### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by NeffIsBack**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:13]: We don't use this - yet. I have a discussion here: https://github.com/Pennyw0rth/NetExec/discussions/23 about it though.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:11]: capitalize GitHub

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Contributing to Transcriptase
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:10]: capitalize GitHub

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. Ensure the test suite passes.
+4. Make sure your code lints.
+5. Issue that pull request!
+
+## Any contributions you make will be under the BSD-2-Clause Software License
+In short, when you submit code changes, your submissions are understood to be under the same [BSD-2-Clause License](https://choosealicense.com/licenses/bsd-2-clause/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/briandk/transcriptase-atom/issues)
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:25]: capitalize GitHub

Comment (Marshall-Hallenbeck) [reply CONTRIBUTING.md:25]: Also the issue link isn't to our repo

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:26]: Link: https://github.com/Pennyw0rth/NetExec/issues/new/choose

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
+[PEP](https://peps.python.org/pep-0008/) is used in this project
+
+* 4 spaces for indentation rather than tabs
```
Comment (Marshall-Hallenbeck) [inline CONTRIBUTING.md:45]: Add "double quotes on outside of strings" as well?

**Review by Marshall-Hallenbeck — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (NeffIsBack) [reply CONTRIBUTING.md:25]: woops 😅 copied the file from some template lol

File: CONTRIBUTING.md (hunk 5 above)
Comment (NeffIsBack) [reply CONTRIBUTING.md:26]: damn, forgot that as well

**Review by Marshall-Hallenbeck — APPROVED** [review]

**Pull request merged**